git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
@jeremywiebe a few additional points I've been thinking of:
- While tracing a code change through commit to PR to JIRA ticket is possible, JIRA ticket numbers in commit messages mean that you have a double-linked paper trail: you can find commits based on a given JIRA ticket.
- While PRs are our new standard of atomic code operations, it means that relying solely on them for uncovering history and context is being locked into GitHub's interface. With JIRA ticket numbers in commit messages, you can search for them in _any_ git client.
- Discipline in commit messages is exactly what I'm looking for with these standards. Getting developers to micro-document their changes through commit messages means less work for maintainers down the line.